config/routes.rb

```diff
@@ -40,6 +40,10 @@
 
   get '/upload' => 'spectrums#new'
   post '/spectrums/create' => 'spectrums#create'
+  get '/spectrums/destroy/:id' => 'spectrums#destroy' 
+  get '/spectrums/:id/edit' => 'spectrums#edit'
+  post '/spectrums/:id/update' => 'spectrums#update'
```

Comment:
```suggestion
  post '/spectrums/:id/update' => 'spectrums#update'
  post '/spectrums/fork/:id' => 'spectrums#fork'
  post '/spectrums/choose/:id' => 'spectrums#choose'
```

Just adding a few more missing ones!!